pkg/storage/emptydir/emptydir.go

```diff
@@ -52,5 +52,11 @@ func (d *driver) Volumes() ([]corev1.Volume, []corev1.VolumeMount, error) {
 }
 
 func (d *driver) CompleteConfiguration(customResourceStatus *opapi.ImageRegistryStatus) error {
+
+	d.Config.VolumeSource = corev1.VolumeSource{
```

Comment:
Is it an intention to modify the custom resource? Why do we need it?

Reply:
Updated, that was leftover from a previous change that got updated.